### PR TITLE
Layout dynamic/slow-mpu Container Without MPU

### PR DESCRIPTION
**Pull request by kelvin-chappell**
Now this container has different layouts according to whether or not it should show an MPU.

Single slice with MPU:
image

Single slice without MPU:
image

Multiple slice with MPU:
image

Multiple slice without MPU:
image

/cc @Pearson82 @janua

### CODE REVIEW

**Comment by Pearson82** [comment]
Layout looks good to me :+1:

File: common/app/views/support/GetClasses.scala [hunk 1]
```diff
@@ -58,7 +58,7 @@ object GetClasses {
       containerDefinition.showLatestUpdate,
       containerDefinition.index == 0 && containerDefinition.customHeader.isEmpty,
       containerDefinition.displayName.isDefined,
-      containerDefinition.displayName == Some("headlines"),
+      containerDefinition.displayName.contains("headlines"),
```
Comment (janua) [inline common/app/views/support/GetClasses.scala]: Won't this fail on `scala 2.10`?
Locally I think this won't compile for me, as `Option` doesn't have a `contain` method before `2.10`.

**Comment by janua** [comment]
:+1:

File: common/app/views/support/GetClasses.scala (hunk 1 above)
Comment (kelvin-chappell) [inline common/app/views/support/GetClasses.scala]: See https://github.com/guardian/frontend/blob/master/project/Prototypes.scala#L26

You're behind the times!

Comment (janua) [inline common/app/views/support/GetClasses.scala]: I'm not sure why, but I have been bitten by this about 3 times now and I **thought** I was on 2.11 but it then fails to compile and had to revert it.

**Pull request merged**